src/app/lab/components/units-dropdown.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+import { Chart } from '../../models/chart.model';
+import { Indicator } from '../../models/indicator.model';
+
+/*  Units Dropdown Component
+
+    -- Requires handling unit selection
+    Expected use:
+        <ccl-units-dropdown
+            [indicator]="your_indicator"
+            [unit]="your_unit"
+            (unitSelected)="onUnitSelected($event)">
+*/
+
+@Component({
+  selector: 'ccl-units-dropdown',
+  template: `<div dropdown class="dropdown dropdown-units">
+              <button dropdownToggle class="button dropdown-toggle" type="button"
+                id="unitsDropdown" data-toggle="dropdown" aria-haspopup="true"
+                aria-expanded="true">
+                <i class="icon-flash"></i>
+                {{ unit || indicator.default_units }}
```

Comment:
If this were a really public tool like a NYT article graphic, human readable would make perfect sense. I don't think this is that audience